ui/README.md

```diff
@@ -0,0 +1,14 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Local Development using VSCode Chrome Debugger
+
+Run:
+
+### `npm start`
```

Comment:
I have tested it but `npm start` shouldn't be needed to run in VS Code.

Reply:
It's needed for now @davidffrench as the `.vscode/launch.json` does not exist yet. I think it should be kept in regardless as not everyone uses VS Code (our team do, but this is a public project so there will be others who want to run this).

Reply:
@davidffrench @craicoverflow with the launch.json in place I'm able to run in debug mode, but the application is unavailable until I run `npm start`. I could be missing a step somehow. The file is checked in now so would you mind checking that on your end, please. 😄

Reply:
@laurafitzgerald just finished testing that. I think that is the expected behaviour as the `launch.json` does not actually start the server, it binds to the URL which the server is running on already, so it needs to be left in. Maybe @davidffrench could confirm.

Reply:
Ye are absolutely right. You need to run `npm start`. https://code.visualstudio.com/docs/nodejs/reactjs-tutorial#_configure-the-chrome-debugger